src/pages/training.js

```diff
@@ -1,15 +1,324 @@
 import React from 'react';
 import Helmet from 'react-helmet';
+import withStyles from 'react-jss';
+import cx from 'classnames';
 
-import { Header } from '../components';
+import letsChat from '../assets/images/footer/lets_chat.png';
+import clients from '../data/clients.json';
+import imagePatternDots from '../assets/images/patterns/dots.png';
+import comprehensive from '../assets/images/training/comprehensive.png';
+import bestPractices from '../assets/images/training/best_practices.png';
+import realWorld from '../assets/images/training/real_world.png';
+import professionalInstructors from '../assets/images/training/professional_instructors.png';
```

Comment:
I don't remember if I used "_" or "-" to separate words for files. Put those file names either with one or the other. Check the other files in other folders. The idea is to have consistency. If you find files in other places with the wrong names, please update them wherever needed.